eos-update-flatpak-repos

```diff
@@ -635,6 +635,26 @@ def _migrate_installed_flatpaks():
             try:
                 new_branch = migrate.get('new-branch')
                 if new_branch:
+                    new_ref = None
```

Comment:
This definition is not needed, the scope of the definition inside `try` is enough and it's not used outside the `else` branch

Reply:
yayaya